huaweicloud/services/ga/resource_huaweicloud_ga_address_group.go

```diff
@@ -52,14 +57,12 @@ func ResourceIpAddressGroup() *schema.Resource {
 			"ip_addresses": {
 				Type:     schema.TypeSet,
 				Optional: true,
-				Computed: true,
-				ForceNew: true,
 				Elem:     ipAddressGroupSchema(),
-				MaxItems: 20,
```

Comment:
we must process the length of ip_addresses > 20 when creating.

Reply:
done